docs/src/components/home/ValueProposition.tsx

```diff
@@ -41,11 +41,10 @@ const ValueProposition = () => {
   return (
     <Container sx={{ py: { xs: 4, sm: 6, md: 8 } }}>
       <Typography variant="body2" color="primary" fontWeight="bold">
-        Why go with us?
+        Developer experience 
       </Typography>
       <Typography variant="h2" sx={{ mt: 1, mb: { xs: 2, sm: 4 }, maxWidth: 500 }}>
-        Used by the world&apos;s best teams, supporting design systems of{' '}
-        <GradientText>all types</GradientText>
+        Powerful tools to help you build{' '}<GradientText>all types</GradientText>{' '}of user interfaces
```

Comment:
@oliviertassinari @mbrookes 

> We're still leaking internal meta-labels. Is it intentional? If the heading doesn't communicate the intention of the section, might it be better to improve the copy.

I never saw that copywriting as _"internal meta-labels"_ but, regardless, I've spent some time today thinking of ways to improve it. This section displays the four main value propositions of the product. It is indeed meant to answer the _"Why should I use this?"_ question that pops in one's head when she/he or team is considering and looking for libraries for the project at hand. 

When I myself was in the position to look for libraries in previous companies, me and my team were looking for libraries that provided two high-level hard things: 
1. One that enabled us to deliver great and custom experiences to users/customers: hence _beautifully designed, easily customized, accessible in mind._
2. While having a nice experience when building as well: hence _superb documentation, easily customized._ 

I wrote this iteration trying to match this rationale. I'm still reflecting if it should have "Developer _and user_ experience" as well... Also thought that this would match what I understand that drove many of the v5 efforts (developer pains, customizability). What are your thoughts, does it makes sense? Let's use this thread to iterate and resolve this.